packages/website/src/components/Footer.tsx

```diff
@@ -80,14 +79,6 @@ export default function Footer() {
           <FaStackOverflow />
           <VisuallyHidden>Stackoverflow</VisuallyHidden>
         </Anchor>
-        <Anchor
-          href="https://spectrum.chat/reakit"
-          target="_blank"
-          onClick={track("reakit.footerSpectrumClick")}
-        >
-          <SpectrumLogo />
-          <VisuallyHidden>Spectrum</VisuallyHidden>
-        </Anchor>
```

Comment:
Let's keep the link on the Footer. There are other links there that aren't on header too.